std/bigint.d

```diff
@@ -910,14 +907,14 @@ public:
         Warning: Casting to/from `const` or `immutable` may break type
         system guarantees. Use with care.
      */
-    T opCast(T)() pure nothrow @nogc const
+    T opCast(T)() pure nothrow @nogc @safe const
```

Comment:
Should this really be `@safe` then?

Reply:
I don't actually know what "type system guarantees" that comment refers to, to me it seems just as `@safe` as casting `immutable int` or `const int` to `int`: a simple copy is made that can be mutated.
Also there's not `@trusted` code involved, so if it compiles it should be good.

Reply:
Why is a template function being marked expliictily as `@safe`?

Reply:
So that when it gets turned into something not `@safe` the error message points out why instead of simply saying " the `@safe` unittest can't call the `@system` template function`.
I mostly did it to be consistent with existing annotations on template functions in std.bigint, but I can do without if that's better.